hash/to_flop.xml

```diff
@@ -14171,7 +14171,7 @@ Thanks to DCMOTO (http://dcmoto.free.fr) for info!
 
 	<!-- can also be found in mo6_flop.xml -->
 	<software name="traitbzzb" cloneof="traitbzz">
-		<description>Traitement Bzzz (Compilation Top Arcade)</description>
+		<description>Traitement Bzzz (compilation Top Arcade)</description>
```

Comment:
I think “Compilation Top Arcade” is the literal name of a collection this was extracted from.